.github/workflows/odoo-ci-cd.yml

```
@@ -41,14 +39,31 @@ jobs:
     steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
-
+        
+      - name: Log in to GHCR
+        uses: docker/login-action@v3
+        with:
+              registry: ghcr.io
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 14

(indentation)